packages/teams-js/src/internal/communication.ts

```diff
@@ -543,6 +694,93 @@ export function sendMessageEventToChild(actionName: string, args?: any[]): void
   }
 }
 
+const tryPolyfillWithNestedAppAuthBridgeLogger = communicationLogger.extend('tryPolyfillWithNestedAppAuthBridge');
+
+/**
+ * @hidden
+ * Attempt to polyfill the nestedAppAuthBridge object on the given window
+ *
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+function tryPolyfillWithNestedAppAuthBridge(window: Window | null): void {
+  const logger = tryPolyfillWithNestedAppAuthBridgeLogger;
+
+  if (!window) {
+    logger('Cannot polyfill nestedAppAuthBridge as current window does not exist');
+    return;
+  }
+
+  const extendedWindow = Communication.currentWindow as unknown as NestedAuthExtendedWindow;
+  const nestedAppAuthBridge = createNestedAppAuthBridge(Communication.currentWindow);
+  if (nestedAppAuthBridge) {
+    extendedWindow.nestedAppAuthBridge = nestedAppAuthBridge;
```

Comment:
Won't this replace an existing `nestedAppAuthBridge` if called in a native context? Is there some check so that this only gets called in web contexts?

Reply:
Yeah, good catch. Let me fix that.